mkdocs.yml

```diff
@@ -133,6 +134,7 @@ nav:
           - The Search Bar: "user-guides/getting-started/search-bar.md"
           - Git as a Data Source: "user-guides/git-data-source.md"
           - GraphQL: "user-guides/graphql.md"
+          - Custom Fields: "user-guides/custom-fields.md"
```

Comment:
Can we please alpha sort the User Guides while you're in here touching this? I think it will help navigation.